integration_tests/test_aggregator_settlement.py

```diff
@@ -0,0 +1,159 @@
+# pylint: disable=too-many-instance-attributes, broad-except
```

Comment:
FYI, backend team had decided to not disable pylint at the whole module level, but only at places where it needs to be.

No need to adapt now, just to keep in mind for future PRS.

Reply:
It depends on the specific warning we're suppressing. For example, if we want to suppress the missing docstring warnings for an entire test module, it might make sense to put it on top rather than repeating it 20 times. If we're silencing just 2 or 3 warnings, then I would recommend placing the disable comment on the same line/scope.